src/components/Forms/Subscribe.tsx

```diff
@@ -92,7 +109,11 @@ class SignUp extends React.Component {
           validationSchema={SubscribeSchema}
           onSubmit={values => this.handleSubmit(values)}
         >
-          {({ errors, touched, isSubmitting }) => (
+          {({
+            // errors,
+            // touched,
```

Comment:
I commented all unused props (if you fell you need to use it later, you can uncomment them)

Reply:
very smart!